docs/source/user_guide.rst

```diff
@@ -16,7 +16,7 @@ To install ``ucc`` run:
    pip install -e . # Editable mode
 
 
-UCC requires Python version :math:`\ge` 3.12. 
+UCC requires Python version ≥ 3.12. 
```

Comment:
No problem from me, but why does this need to change? Seems to be rendering fine: https://ucc.readthedocs.io/en/latest/user_guide.html

Reply:
Yep RTD is fine either way, but it didn't render on in the Markdown preview on our GitHub 😐